### DISCUSSION TITLE
Spectre Importer Install Issue

### DISCUSSION THREAD

**Discussion by daprofessa19** [firefly-iii/firefly-iii#5273] [Firefly III]
Hi,

I'm revisiting an issue that I am having with installing the Spectre Importer. I have updated php on my system to the latest version, and I'm using the documentation to install it via composer (I'm not very familiar with composer).

I have looked at many examples, and I am trying to tell composer to use a specific php version, but I am still getting the below error.

[InvalidArgumentException]
  Could not find package firefly-iii/spectre-importer with version 2.2.1 in a version installable using your PHP version, PHP
   extensions and Composer version.

Any help with troubleshooting would be appreciated.

**Comment by JC5** [comment]
What's the output of php -v?

**Comment by daprofessa19** [comment]
Ok, so php points to PHP 7.3.31, but I also have php7.4 and php8.0 available on the machine. 

And in my command, I am explicitly telling composer to use one of these versions instead to create the project.

I can't change the php alias to one of these newer versions because it might break other apps on this server. But from some research, it looks like you can force composer to use a different version of php. Im just not familiar with composer.

**Comment by daprofessa19** [comment]
I have a question. What is the difference between creating the project via the composer command line and for instance, just going to the Github code page and then copying that entire directory to the web server?

Would that work the same?

**Reply by JC5** [reply]
It's basically what composer does. I'm just not sure how to make sure it points to php 8.0
